test/controllers/users_controller_test.rb

```diff
@@ -438,35 +438,53 @@ def change_user_password(user_factory, new_password)
         assert_match(/You do not have permission to perform this action./, flash[:alert])
       end
 
+      should "can give permissions to all applications even where signin is not delegatable or they don't have access to it" do
```

Comment:
Is it worth making the distinction between the admin and the person being granted rights by saying:
`can give permissions to a user for all applications in signon, even when signin is not delegatable or the admin does not have access to those applications`
?